kingpin/actors/aws/test/integration_s3.py

```diff
@@ -27,8 +27,8 @@ class IntegrationS3(testing.AsyncTestCase):
     Requirements:
         You have to create an S3 Bucket named kingpin-integration-test and
         place it in the specified region (default us-east-1).
-        As with other tests, environment variables AWS_ACCESS_KEY_ID and
-        AWS_SECRET_ACCESS_KEY are expected, and the key should have
+        As with other tests, environment variables AWS_ACCESS_KEY_ID,
+        AWS_SECRET_ACCESS_KEY and AWS_SESSION_TOKEN are expected, and the key should have
```

Comment:
Nit missing comma.

Reply:
updated verbiage

        You must have an AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY if using
        an IAM user/role and also an AWS_SESSION_TOKEN if using temporary access credentials,
        with permissions to